.circleci/config.yml

```diff
@@ -196,7 +196,7 @@ jobs:
 
   macos:
     macos:
-      xcode: "10.0.0"
+      xcode: "12.0.0"
```

Comment:
Do we need this at all, since it is set as default above (same for some other settings like TERM)?

Reply:
Definitely seems like we should be able to leverage that indeed. Before I forget this and the issue mentioned in the OP, I filed a ticket https://github.com/facebook/hermes/issues/353